verbs_examples/uniswap/sim.py

```diff
@@ -157,8 +157,6 @@ def init_cache(
     block_number: int,
     seed: int,
     n_steps: int,
-    mu: float = 0.1,
-    sigma: float = 0.6,
```

Comment:
How come these are not needed now?

Reply:
The DummyUniswapAgent is exploring a range of ticks one by one, so in the initial simulation the Gbm is not involved and mu and sigma are not needed.